src/IssueViz.UnitTests/Editor/IssueSpanCalculatorTests.cs

```diff
@@ -0,0 +1,302 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.Text;
+using Moq;
+using SonarLint.VisualStudio.Core.Helpers;
+using SonarLint.VisualStudio.Integration.UnitTests;
+using SonarLint.VisualStudio.IssueVisualization.Editor;
+
+namespace SonarLint.VisualStudio.IssueVisualization.UnitTests.Editor
+{
+    [TestClass]
+    public class IssueSpanCalculatorTests
```

Comment:
Class was renamed and a few tests added, not sure why the diff is not shown correctly.

Reply:
The poor diff experience makes it very difficult to tell which tests are the new tests, particularly since it looks like you've renamed the existing tests. 
In future PRs it would help if you called out the new tests so reviewers could focus on them, rather than having to review the whole file.

Reply:
I'm guessing the diff doesn't show correctly because you've made so many changes to the existing tests that git doesn't recognise it as a renamed file. The volume of change effectively means the whole file has to be reviewed.